sunpy/physics/transforms/limb_darkening_correct.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+
+__author__ = ["Norbert Gyenge"]
+
+
+def limb_darkening_correct(observation, limb_cut=True):
+    """Calculate limb darkening function for specified wavelength
+    and use to remove limb darkening effects from input image.
+    The limb darkening function uses a 5th ordern polynomial
+    fitting to the limb darkening constants obtained from
+    Astrophysical Quantities by Allan. Only in the wavelength
+    range between 4000 and 15000 A.
+
+    Parameters
+    ----------
+        observation - SunPy Map object
+            SDO observation
+
+        limb_cut - Default: True
+            Background (r>1) values will be repleaced by np.nan
+
+    Returns
+    -------
+        observation - Corrected map
+
+    Notes
+    -----
+        IDL code equavalent:
+        http://hesperia.gsfc.nasa.gov/ssw/packages/nrl/idl/nrlgen/analysis/limb_dark.pro
+
+    Reference
+    ---------
+        Cox, Arthur N., Allen's astrophysical quantities, 4th ed. Publisher:
+            New York: AIP Press; Springer, 2000. ISBN: 038798746
+
+    Examples
+    --------
+        >>> myfit = sunpy.map.Map('test.fits')
+        >>> myfit_c = limb_darkening_correct(myfit, limb_cut=True)"""
+    
+    wavelength = observation.meta['WAVELNTH']
+    x_center = observation.meta['crpix1']
+    y_center = observation.meta['crpix2']
```

Comment:
I assume here what you want is the pixel value for the centre of the Sun, which is always the 'CRPIX' so what you want is:

```python
x_centre, y_centre = map.data_to_pixel((0,0)*u.arcsec)
```